integration-tests/src/test/java/com/linkedin/kafka/clients/largemessage/LargeMessageIntegrationTest.java

```diff
@@ -116,6 +115,20 @@ public void onCompletion(RecordMetadata recordMetadata, Exception e) {
         consumptionStarted = true;
       }
       for (ConsumerRecord<String, String> consumerRecord : records) {
+        // Verify headers
+        Map<String, byte[]> headers = LiKafkaClientsTestUtils.convertHeaders(consumerRecord.headers());
+        assertTrue(headers.containsKey(Constants.TIMESTAMP));
+        assertEquals(8, headers.get(Constants.TIMESTAMP).length);
+        assertTrue(
+            PrimitiveEncoderDecoder.decodeLong(headers.get(Constants.TIMESTAMP), 0)
+                > System.currentTimeMillis() - TimeUnit.DAYS.toMillis(1)
```

Comment:
Can we record start and end timestamps of producer.send() above and make sure the timestamp in the header is within that range?

Reply:
+1, better bound than 1 day